good-enough-practices-for-scientific-computing.tex

```diff
@@ -580,7 +580,7 @@ \section{Collaboration}\label{sec:collaboration}
   doi={10.1371/journal.pone.0065848}
 }
 \end{verbatim}
-\caption{Example CITATION File}
+\caption{Example CITATION File with the information in two forms, as a regular citation and in BiBTeX format\fixme{LN}{explain BiBTex?}}
```

Comment:
Thats was my first thought, but then I realised many of our readers will not recognise the second format...